e2e/utils/command-utils.ts

```diff
@@ -279,10 +279,7 @@ export function runNgAdd(
   try {
     const pmc = getPackageManagerCommand();
     packageInstall(packageName, undefined, version);
-    const fullCommand = pmc.run(
-      `ng g ${packageName}:ng-add`,
-      `${command}${isVerbose() ? ' --verbose' : ''}` ?? ''
-    );
+    const fullCommand = pmc.run(`ng g ${packageName}:ng-add`, command ?? '');
```

Comment:
unsure if this is supposed to be removed or just updated

Reply:
We want `verbose` on nightly to debug failed tests more easily, but if CLI doesn't support such flag it can break (e.g. git commands)